stories/ReactVegaLiteDemo.jsx

```diff
@@ -70,7 +72,7 @@ export default class Demo extends React.Component {
 
   handleHover(...args) {
     this.setState({
-      info: JSON.stringify(args),
+      info: JSON.stringify(args)
```

Comment:
`yarn lint:fix`

Reply:
Seems like the whole file has been reformatted with double quote and different `eslint` rule.